packages/storefront-events-collector/src/contexts/searchResults.ts

```diff
@@ -26,6 +26,8 @@ const createContext = (searchUnitId: string, searchResults?: SearchResults): Sea
         data: {
             searchUnitId: searchResultsUnit.searchUnitId,
             searchRequestId: searchResultsUnit.searchRequestId,
+            rankingType: searchResultsUnit.rankingType,
+            trendingWindow: searchResultsUnit.trendingWindow,
```

Comment:
i know you are just adding these two fields but you could spread this over the data attribute, and then map the searchResults that don't match the data type like so
```ts
        data: {
            ...searchResultsUnit
            productCount: searchResultsUnit.products.length,
            categoryCount: searchResultsUnit.categories.length,
            suggestionCount: searchResultsUnit.suggestions.length,
        },
```